flexget/plugins/output/pushbullet.py

```diff
@@ -45,31 +47,26 @@ class OutputPushbullet(object):
     @plugin.priority(0)
     def on_task_output(self, task, config):
 
-        # Support for multiple devices
-        devices = config.get('device')
+        devices = config.get('device', [])
         if not isinstance(devices, list):
             devices = [devices]
 
-        # Support for urls
-        push_type = 'link' if config.has_key('url') else 'note'
+        emails = config.get('email', [])
+        if not isinstance(emails, list):
+            emails = [emails]
 
-        # Set a bunch of local variables from the config
-        apikey = config['apikey']
+        destinations = []
+        for typ, val in [('device_iden', devices), ('email', emails)]:
+            destinations.extend([(x, typ) for x in val])
```

Comment:
This is a bit confusing, I think it might be a bit more clear just making a separate loop for each type.